beacon/manager.go

```diff
@@ -264,23 +338,26 @@ func (s *Manager) processNewSlot(ctx context.Context, state State, client Beacon
 		}(received)
 	}
 
-	// query expected withdrawals root
-	go s.updatedExpectedWithdrawals(headSlot, state, client)
+	if !s.Config.RunPayloadAttributesSubscription {
```

Comment:
I wonder shouldn't we allow that regardless of payload atributes, like what if payload attributes fails and the other one wont. can we hook it up somehow to our manual requesters?